Gems/PhysX/Code/Source/Joint/PhysXJointUtils.cpp

```diff
@@ -224,8 +224,22 @@ namespace PhysX {
                 physx::PxJointLimitCone limitCone(swingLimitY, swingLimitZ);
                 joint->setSwingLimit(limitCone);
 
-                const float twistLower = AZ::DegToRad(AZStd::GetMin(configuration.m_twistLimitLower, configuration.m_twistLimitUpper));
-                const float twistUpper = AZ::DegToRad(AZStd::GetMax(configuration.m_twistLimitLower, configuration.m_twistLimitUpper));
+                float twistLower = AZ::DegToRad(AZStd::GetMin(configuration.m_twistLimitLower, configuration.m_twistLimitUpper));
+                float twistUpper = AZ::DegToRad(AZStd::GetMax(configuration.m_twistLimitLower, configuration.m_twistLimitUpper));
+                // make sure there is at least a small difference between the lower and upper limits to avoid problems in PhysX
+                const float minSwingLimitRangeRadians = AZ::DegToRad(JointConstants::MinTwistLimitRangeDegrees);
```

Comment:
would a more appropriate name here be minTwistLimitRangeRadians?

Reply:
yeah, you're right, stupid typo